src/main/java/core/basesyntax/Command.java

```diff
@@ -0,0 +1,130 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+/**
+ * Right input: create C:\Users\Lenovo\Desktop\ test.txt
+ * Wrong input: create C:\Users\Lenovo\Desktop test.txt
+ */
+public class Command {
+    private static final DateTimeFormatter FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yy HH:mm:ss");
+    private Scanner scanner;
+
+    public Command(Scanner scanner) {
+        this.scanner = scanner;
+    }
+
+    public void createFile(String path, String fileName) throws IOException {
```

Comment:
can we simplify it using exists() method?

Reply:
I don't know how i can simplify it more. I made it a little more readable but i don't know how i can simplify it with exists() method. I need to call this method twice because first time we check if entered path exists and then check if entered file name exists and only after those two checks if they passed we can create a new file.

Reply:
I can't simplify it with exists() method. I did it a little bit readable but I need to call exists method twice because according to README.md I need to check if path exists and if not print that such path does not exist, after that I need to check if such file exists and if true to ask user if he wants to rewrite this file or not. And only after those two checks if both of them were false I can create file with such path and file name.